src/algokit/core/constants.py

```diff
@@ -0,0 +1,89 @@
+# mypy: disable-error-code="typeddict-unknown-key"
```

Comment:
hmmm what's the rational for disabling this mypy check? can we structure the code so that it passes without it instead?

Reply:
There was some strange errors with mypy conflicting with ruff when using the union operator is the algorand networks, i've split them into 2 separate dics that are merged at the end, should be fixed now